spec.html

```diff
@@ -21128,6 +21240,34 @@ <h1>Static Semantics: Early Errors</h1>
       <ul>
         <li>It is a Syntax Error if StringValue of |PrivateIdentifier| is *"#constructor"*.</li>
       </ul>
+
+      <emu-grammar>ClassStaticBlockBody: ClassStaticBlockStatementList</emu-grammar>
+      <ul>
+        <li>
+          <p>It is a Syntax Error if the LexicallyDeclaredNames of |ClassStaticBlockStatementList| contains any duplicate entries.</p>
```

Comment:
```suggestion
          It is a Syntax Error if the LexicallyDeclaredNames of |ClassStaticBlockStatementList| contains any duplicate entries.
```
(and likewise below)